platform/broadcom/sai.mk

```diff
@@ -4,6 +4,11 @@ BRCM_SAI_DEV = libsaibcm-dev_4.3.3.5-1_amd64.deb
 $(eval $(call add_derived_package,$(BRCM_SAI),$(BRCM_SAI_DEV)))
 $(BRCM_SAI_DEV)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/4.3/master/libsaibcm-dev_4.3.3.5-1_amd64.deb?sv=2019-12-12&st=2021-04-26T22%3A17%3A56Z&se=2030-04-27T22%3A17%3A00Z&sr=b&sp=r&sig=2eWNoczh9P5c3Ir%2B1YL08ZkPol9GUkvNJKFGf%2BacXlI%3D"
 
+# SAI module for DNX Asic family
+BRCM_DNX_SAI = libsaibcm_dnx_5.0.0.1_amd64.deb
+$(BRCM_DNX_SAI)_URL = "https://sonicstorage.blob.core.windows.net/packages/bcmsai/5.0/master/libsaibcm_dnx_5.0.0.1_amd64.deb?sv=2015-04-05&sr=b&sig=iUW4ZSz43oeOSe21%2BFaNTG1phTr6qgAfeeEN2mCXBWU%3D&se=2035-01-17T05%3A15%3A51Z&sp=r"
+
```

Comment:
I expected to see BRCM_DNX_SAI_DEV for the development headers that are used by the platform/broadcom/rules.mk:$(SYNCD)_DEPENDS.  Which libsaibcm-dev pkg are we injecting into the syncd build?

Reply:
syncd actually needs the SAI headers. The SAI header files are same in both libsaibcm-dev & libsaibcm-dev_dnx package. 

We are making a common syncd*deb package, different SAI packages are included in respective dockers 

platform/broadcom/docker-syncd-brcm-dnx.mk
```
$(DOCKER_SYNCD_DNX_BASE)_DEPENDS += $(SYNCD)
$(DOCKER_SYNCD_DNX_BASE)_DEPENDS += $(BRCM_DNX_SAI)
```
platform/broadcom/docker-syncd-brcm.mk
```
$(DOCKER_SYNCD_DNX_BASE)_DEPENDS += $(SYNCD)
$(DOCKER_SYNCD_DNX_BASE)_DEPENDS += $(BRCM_SAI)
```